src/hotspot/share/runtime/os.inline.hpp

```diff
@@ -27,6 +27,31 @@
 
 #include "runtime/os.hpp"
 
+#include OS_HEADER(os)
 #include OS_HEADER_INLINE(os)
```

Comment:
We usually only explicitly include .inline.hpp and don't and let that file bring in the corresponding .hpp file.